justfile

```diff
@@ -8,6 +8,9 @@ run:
 build_release_linux version="latest":
     GOOS=linux CGO_ENABLED=1 GO111MODULE=on go build -ldflags "-s -w -X 'main.buildVersion={{version}}' -X 'main.buildHash=$(git rev-parse HEAD)' -X 'main.buildDate=$(date -u)'" -trimpath -o dist/kanata-tray
 
+build_release_darwin version="latest":
+    GOOS=darwin GOARCH=amd64 CGO_ENABLED=1 GO111MODULE=on go build -ldflags "-s -w -X 'main.buildVersion=latest' -X 'main.buildHash=$(git rev-parse HEAD)' -X 'main.buildDate=$(date -u)'" -trimpath -o dist/kanata-tray
```

Comment:
Please remove `GOARCH=amd64` if we also want to support x86 macs. Without it, a correct architecture will be selected automatically for current platform.

Reply:
Sure, I can do that.